core/src/main/java/com/graphhopper/reader/dem/AbstractTiffElevationProvider.java

```diff
@@ -1,14 +1,14 @@
 /*
  *  Licensed to GraphHopper GmbH under one or more contributor
- *  license agreements. See the NOTICE file distributed with this work for 
+ *  license agreements. See the NOTICE file distributed with this work for
```

Comment:
I tried to copy the copyright information from a different class in GraphHopper and I think that the applied formatting is actually correct and that the formatting before has been wrong, so I would keep it that way.